app/src/main/res/layout/onlinefeedview_activity.xml

```diff
@@ -64,103 +66,67 @@
                         android:layout_width="0dp"
                         android:layout_height="wrap_content"
                         android:layout_alignParentTop="true"
-                        android:layout_marginBottom="16dp"
-                        android:layout_marginLeft="16dp"
-                        android:layout_marginStart="16dp"
-                        android:layout_marginTop="16dp"
-                        android:layout_marginRight="16dp"
-                        android:layout_marginEnd="16dp"
                         android:layout_alignParentEnd="true"
                         android:layout_alignParentRight="true"
-                        android:layout_toRightOf="@id/coverImage"
+                        android:layout_marginStart="16dp"
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginTop="25dp"
+                        android:layout_marginEnd="26dp"
+                        android:layout_marginRight="26dp"
+                        android:layout_marginBottom="8dp"
                         android:layout_toEndOf="@id/coverImage"
+                        android:layout_toRightOf="@id/coverImage"
                         android:ellipsize="end"
+                        android:lineHeight="20dp"
                         android:maxLines="2"
                         android:shadowColor="@color/black"
                         android:shadowRadius="3"
                         android:textColor="@color/white"
-                        style="@style/AntennaPod.TextView.Heading"
+                        android:textFontWeight="800"
+                        style="@style/AntennaPod.TextView.HeadingSemiBold"
```

Comment:
Font weight can be in the style

Reply:
It doesnt apply, so i had to go this way!